### PR TITLE
Adds Tests For Fluent Divider Web Component 

### PR DESCRIPTION
**Pull request by halerankin**
Adds Playwright tests for Fluent Divider web component.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b03b40647507909a151f966be6cd67b65cfcf3f9'>b03b40647507909a151f966be6cd67b65cfcf3f9</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/divider/divider.spec.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import { expect, test } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { fixtureURL } from '../helpers.tests.js';
+import type { Divider } from './divider.js';
+import { lightModeStylesheetBehavior } from '@microsoft/fast-foundation';
```
Comment (halerankin) [inline packages/web-components/src/divider/divider.spec.ts:5]: I plan to remove this.

Comment (halerankin) [reply packages/web-components/src/divider/divider.spec.ts]: Removed

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**